geometric/prepare.py

```diff
@@ -183,7 +183,7 @@ def get_molecule_engine(**kwargs):
             M.mult = tcin.get('spinmult',1)
             # The TeraChem engine needs to write rst7 files before calling TC
             # and also make sure the prmtop and qmindices.txt files are present.
-            engine = TeraChem(M, tcin, dirname=dirname)
+            engine = TeraChem(M[-1], tcin, dirname=dirname)
```

Comment:
Why are we using the last frame in the Molecule object specifically?

Reply:
When TeraChem is used for NEB, its input file points at the xyz file with multiple molecules. This leads to the error in `engine.py`: [https://github.com/leeping/geomeTRIC/blob/master/geometric/engine.py#L195](https://github.com/leeping/geomeTRIC/blob/master/geometric/engine.py#L195)

I picked the last frame based one this: [https://github.com/leeping/geomeTRIC/blob/master/geometric/params.py#L339](https://github.com/leeping/geomeTRIC/blob/master/geometric/params.py#L339)